app/forms/placements/placements/filter_form.rb

```diff
@@ -14,15 +14,15 @@ def initialize(params = {})
   end
 
   def filters_selected?
-    attributes.values.compact.flatten.any?
+    attributes.except("placements_to_show").values.compact.flatten.any?
   end
 
   def clear_filters_path(search_location: nil)
-    placements_placements_path(search_location:)
+    placements_placements_path(search_location:, filters: { placements_to_show: })
```

Comment:
Would the expected user behaviour be that clicking "Clear filters" resets the 'Placements to show' filter to its default value?

Right now it looks like the selected option will be persisted. I'm wondering if it should actually revert to the default.

```suggestion
    placements_placements_path(search_location:, filters: { placements_to_show: "available_placements" })
```

Reply:
This was deliberate, if we don't show the pills or option to clear filters regardless of the state of this selection, I don't think it makes sense to have it change that filter if something else has been added.